advanced_new_file/completions/fuzzy_sort.py

```diff
@@ -1,15 +1,19 @@
 from operator import itemgetter
 # import numpy as np
 
-def sort_by_fuzzy(query, choices):
+def sort_by_fuzzy(query, choices, limit:int = 0):
```

Comment:
Adding a type hint requires Python 3.6 syntax, meaning the package becomes restricted to ST4 and requires a `.python-version` file. Which is fine in general, but needs to be handled accordingly.

Reply:
Annotations alone are not a problem, as long as one is not using the `typing` module itself and the types that it exports.

That said, without the `typing` module it won't be possible to properly annotate everything so not sure if there is point in partial annotations like that.